.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can you add documentation for the configuration into `docs/tools/prettier.md`?

Reply:
Done